### PR TITLE
Made supported

### PR DESCRIPTION
**Pull request by scott1702**
This change bring the module's docs up to a 'supported' standard (as defined at http://www.silverstripe.org/software/addons/supported-modules-definition/)

Docs added:
- .editorconfig
- .gitattributes
- contributing.md
- license.md

Docs changed:
- .travis.yml
- README.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+env:
+  - DB=MYSQL CORE_RELEASE=3.1
+
+matrix:
+  include:
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.1 SECUREASSETS=1
```
Comment (tractorcow) [inline .travis.yml]: We don't need to test this in this module.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - 5.6
+  - 7.0
+
+env:
+  - DB=MYSQL CORE_RELEASE=3.1
+
+matrix:
+  include:
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3
```
Comment (tractorcow) [inline .travis.yml]: Please add a CORE_RELEASE=3.2 as well.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,12 @@
 This module adds a "All content, page and files from across all subsites" report in the CMS, so that
 an administrator can get a quick overview of content across subsite in the site.
 
+[![Build Status](http://img.shields.io/travis/silverstripe/sitewidecontent-report.svg?style=flat-square)](https://travis-ci.org/silverstripe/silverstripe-userforms)
+[![Code Quality](http://img.shields.io/scrutinizer/g/silverstripe/sitewidecontent-report.svg?style=flat-square)](https://scrutinizer-ci.com/g/silverstripe/silverstripe-userforms)
```
Comment (tractorcow) [inline README.md]: You're linking to userforms here?

File: composer.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
 	],
 	"type": "silverstripe-module",
 	"require": {
+		"silverstripe/framework": "~3.1",
```
Comment (assertchris) [inline composer.json]: Why is this required? `cms` requires `framework`.

Comment (scott1702) [inline composer.json]: duh, good point

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.2
+
+before_script:
+  - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
+  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
+  - cd ~/builds/ss
+
+script:
+  - vendor/bin/phpunit --coverage-clover coverage.clover sitewidecontent-report/tests
```
Comment (assertchris) [inline .travis.yml]: Please remove coverage reporting - it's clearly not working, and isn't technically required by the standard.

File: README.md [hunk 6]
```diff
@@ -4,6 +4,12 @@
 This module adds a "All content, page and files from across all subsites" report in the CMS, so that
 an administrator can get a quick overview of content across subsite in the site.
 
+[![Build Status](http://img.shields.io/travis/silverstripe/sitewidecontent-report.svg?style=flat-square)](https://travis-ci.org/silverstripe/sitewidecontent-report)
+[![Code Quality](http://img.shields.io/scrutinizer/g/silverstripe/sitewidecontent-report.svg?style=flat-square)](https://scrutinizer-ci.com/g/silverstripe/sitewidecontent-report)
+[![Code Coverage](http://img.shields.io/scrutinizer/coverage/g/silverstripe/sitewidecontent-report.svg?style=flat-square)](https://scrutinizer-ci.com/g/silverstripe/sitewidecontent-report)
```
Comment (assertchris) [inline README.md]: Can probably remove this badge now... :)
